migrations/2023-08-31-205559_add_image_upload/up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE image_upload (
+    id serial PRIMARY KEY,
+    local_user_id int REFERENCES local_user ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
+    pictrs_alias text NOT NULL,
```

Comment:
See comment above, but this table should probably store not the aliases, but the unique internal pictrs ids, then this column made unique.

Reply:
mentioned in other comment: this column can be made unique as-is. aliases are unqiue.